.github/stale.yml

```diff
@@ -0,0 +1,36 @@
+# Configuration for Probot (Stale) - https://github.com/probot/stale
+
+# Ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Label to use when marking as stale
+staleLabel: archived
+
+# Limit the number of actions to 1 per hour
+limitPerRun: 1
+
+# Do not archive issues that require a SEMVER major release, those normally
+# have a longer lifeycle.
+exemptLabels:
```

Comment:
I wonder if we should exempt any issue with labels indicating that the issue has already been groomed. The most likely issues to be stale are the bug reports we haven’t even looked at that are quite old.

Reply:
Could you share an example or two? In principle, I'm also interested in archiving issues that had no recent activity even if someone looked at them at some point.